app/views/users/show.html.erb

```diff
@@ -2,16 +2,23 @@
   <div class="col-md-5 col-md-offset-1">
     <div class="card card--user">
       <div class="user-header">
-        <h1 class="text-center"><%= @user.username %></h1>
+        <h1 class="text-center">@<%= @user.username %></h1>
```

Comment:
if you are going to show always the username with an `@` in front, would be better to create a helper method instead of duplicating the logic everywhere :)